Langs/0407.lang

```diff
@@ -57,3 +57,15 @@ Invalid Release Tags Received!=Ungültige Release-Tags erhalten!
 Invalid Release Types Received!=Ungültige Release-Types erhalten!
 Update Available=Update verfügbar
 An Update is Available, would you like to download it?=Ein Update ist verfügbar. Möchten Sie es herunterladen?
+[Descriptions]
+Architecture=Die Datenmenge, die Ihre CPU und Ihr Betriebssystem gleichzeitig verarbeiten können. Sollten Sie ein 32-Bit-Betriebssystem nutzen, erfordert dies eine 64-Bit Neuinstallation von Windows 11 (Ihre Daten gehen dabei verloren!). 32-Bit-CPU erfordert eine neue CPU.
+Boot=Eine Methode, die Ihr Motherboard zum Booten von Windows verwendet. "Legacy" bedeutet, dass ihr System nicht im UEFI-Modus läuft. Dies kann in den BIOS/UEFI-Einstellungen korrigiert werden. Schlagen Sie im Handbuch Ihres Motherboards nach.
+CPU Name=Die CPU, die Sie in Ihrem Computer installiert haben. Die Kompatibilität kann sich ändern. Bei inkompatibilität, muss die CPU bei Desktop PC's ersetzt werden. Dies ist meistens bei Laptops nicht möglich.
```

Comment:
```suggestion
CPU Name=Die CPU, die Sie in Ihrem Computer installiert haben. Die Kompatibilität kann sich ändern. Bei inkompatibilität, muss die CPU bei Desktop PC's ersetzt werden. Dies ist meistens bei Laptops nicht möglich.
```

@rcmaehl , @botflakes 
Note: On older bords the new CPU doesn't fit because of the changed chipset. We should write this.

Reply:
The texts are already too long. @rcmaehl We need line breaks there I guess